spec/models/user_spec.rb

```diff
@@ -40,54 +40,109 @@
   end
 
   describe 'is_admin' do
+    def expect_admin(value)
+      expect(subject.is_admin).to be value
+      expect(subject.is_admin?).to be value
+    end
     it 'should be true if the group is an admin group' do
       allow(subject).to receive(:groups).and_return(['workgroup:sdr:administrator-role'])
-      expect(subject.is_admin).to be_truthy
+      expect_admin true
     end
-    it 'should be true if the group is a deprecated admin group' do
+    it 'should be false if the group is a deprecated admin group' do
       allow(subject).to receive(:groups).and_return(['workgroup:dlss:dor-admin'])
-      expect(subject.is_admin).to be_truthy
+      expect_admin false
     end
-    it 'should be false otherwise' do
+    it 'should be false for a blank group' do
       allow(subject).to receive(:groups).and_return([])
-      expect(subject.is_admin).to be_falsey
+      expect_admin false
+      allow(subject).to receive(:groups).and_return(nil)
+      expect_admin false
+    end
+    it 'should be false with an inadequate group membership' do
+      allow(subject).to receive(:groups).and_return(['workgroup:dlss:not-admin'])
+      expect_admin false
+    end
+    it 'should be true for ADMIN_GROUPS' do
+      allow(subject).to receive(:groups).and_return(User::ADMIN_GROUPS)
+      expect_admin true
+    end
+    it 'should be false for MANAGER_GROUPS' do
+      allow(subject).to receive(:groups).and_return(User::MANAGER_GROUPS)
+      expect_admin false
     end
-    it 'should be false even with an inadequate group membership' do
-      allow(subject).to receive(:groups).and_return(['workgroup:dlss:not_an_admin'])
-      expect(subject.is_admin).to be_falsey
-      expect(subject.is_admin?).to be_falsey
+    it 'should be false for VIEWER_GROUPS' do
+      allow(subject).to receive(:groups).and_return(User::VIEWER_GROUPS)
+      expect_admin false
     end
   end
 
   describe 'is_manager' do
+    def expect_manager(value)
+      expect(subject.is_manager).to be value
+      expect(subject.is_manager?).to be value
+    end
```

Comment:
to me, this seems like overkill, and i find that it makes the below tests a bit harder to read.  the equivalence of `is_manager` and `is_manager?` is due to an alias.  it seems fine to me to trust that `alias` works as advertised.  if you wanted to be really paranoid, it seems preferable to test once that calling `is_manager?` just calls `is_manager`, rather than calling both each time in a wrapper method.